### ISSUE TITLE
Inline buttons not working

### ISSUE THREAD

**Issue by rlrigdon** [shabegom/buttons#93]
Followed instructions on creating buttons using Button maker. Then used command for creating an inline button, selected the desired button from the dropdown list and the correct code is inserted at the spot. When you go to preview mode, however, you just get an Undefined button with no action attached.

**Comment by ingewertwijn** [comment]
This happens to me also, but only when there is more than one button with the same name. Unfortunately the plugin does not accept characters like '-' in the name, so I had to change some of the button names. But once the duplicates were out, it worked.

**Comment by rlrigdon** [comment]
I have neither of those situations - all of my buttons have unique names and there are no other characters besides letters. (I initially just let the plug assign the id's with the auto-generated names of random characters. Thinking that was the problem, I deleted all of the buttons and then recreated them with my own names - again using just letters - and I still get the same thing.) The insert inline button command lets me choose from the list of buttons defined, the correct shortcode within the tic marks appears, but when you go to preview mode, you just get a button that says "Undefined."  I tried changing themes but it doesn't seem to matter

**Comment by Treygec** [comment]
Not sure if it's related, but in Live Preview I don't have a button being rendered. At the moment, I only have 1 button which contains no non-alphabetic characters. When I place the button with the inline button command I just see the text 'button-getEvents' (getEvents is my button id). However, in reading mode it renders as a button just fine and the button works.

**Comment by shabegom** [comment]
Thanks for reporting. Inline buttons need to be updated to work with Live Preview as it uses a completely different rendering method. Stay tuned.

**Comment by gncorbin** [comment]
I am having this same problem: more than one button inline, the first is "undefined", while the second inline button works. The buttons have unique names. If I add a third button, the previously working button then shows up as "undefined" (first two now undefined). 

At one point, I did get two buttons working, but I cannot, for the life of me, figure out what I did differently.

I am running Obsidian v0.13.19 and Buttons v0.4.16.
